_includes/docs/latest/setup-script-credentials-gke.md

```diff
@@ -2,6 +2,20 @@
 . ./helpers/setup.sh -p PROJECT_ID -o ORG_ID -k
 ```
 
-This will create a service account called `cloud-foundation-forseti-<suffix>`,
-assign it the proper roles, and download the service account credentials to
-`${PWD}/credentials.json`.
+This will create a service account called `cloud-foundation-forseti-<suffix>`.
+Grant the newly created service account the following roles:
```

Comment:
The [setup script already grants the container.admin, networkAdmin, and projectIamAdmin roles](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh#L244). It doesn't hurt to list here, but perhaps we should just add the other roles to the setup script instead of asking the user to run a script and then do some manual work? The container API could also be added to the setup script.

Reply:
Submitted [PR 498](https://github.com/forseti-security/terraform-google-forseti/pull/498) to automate the other roles and API.